apps/agent/global.d.ts

```
@@ -0,0 +1,17 @@
+	id: string;
+}
+interface Window {
+	process: any;
+	require: any;
+	variableGlobal: {
+		API_BASE_URL: string,
+		IS_INTEGRATED_DESKTOP: boolean
+	} | any
+}
+
+declare module NodeJS {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)